BUILD

```diff
@@ -427,6 +427,22 @@ grpc_cc_library(
     ],
 )
 
+grpc_cc_library(
+    name = "grpc++_alts",
+    srcs = [
+        "src/cpp/common/alts_context.cc",
+    ],
+    hdrs = [
+        "include/grpcpp/alts_context.h",
```

Comment:
Note by only changing this file, you are only adding this library for the users building with bazel. If they build with make, they will not be able to use this library. Not sure whether this is intended.

Reply:
Thanks for the reminder Yang! I was not aware of this before. I added some lines to build.yaml and committed all the generated files here. Could you please take a second look?